src/Features/CSharp/Portable/UseLocalFunction/CSharpUseLocalFunctionCodeFixProvider.cs

```diff
@@ -100,12 +106,41 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             editor.ReplaceNode(root, currentRoot);
         }
 
+        private static bool MakeStatic(
+            SemanticModel semanticModel,
+            bool makeStaticIfPossible,
+            LocalDeclarationStatementSyntax localDeclaration,
+            CancellationToken cancellationToken)
+        {
+            // Determines if we can make the local function 'static'.  We can make it static
+            // if the original lambda did not cpature any variables (other than the local 
```

Comment:
>cpature [](start = 46, length = 7)

typo

Reply:
intentional.  it verified you're paying attention

Reply:
Fixed in the useSimpleUsingDeclaration branch.